desktop/src/main/java/bisq/desktop/main/overlays/windows/ManualPayoutTxWindow.java

```diff
@@ -544,6 +553,8 @@ private void calculateTxFee() {
                     .subtract(getInputFieldAsCoin(buyerPayoutAmount))
                     .subtract(getInputFieldAsCoin(sellerPayoutAmount));
             txFee.setText(txFeeValue.toPlainString());
+            double feePercent = (double) txFeeValue.value / getInputFieldAsCoin(amountInMultisig).value;
```

Comment:
Maybe safer to wrap it into a try catch? E.g. if getInputFieldAsCoin(amountInMultisig).value can be 0 it would throw an exception.

Reply:
No exception, it returns `-Infinity`.  So I don't see any issue.

![image](https://user-images.githubusercontent.com/47253594/105407232-0ee4fa80-5bf3-11eb-8b81-5ffd4f9d9cbd.png)